source/coding-the-legislation/35_periods.md

```diff
@@ -149,7 +149,7 @@ You can generate any period with the following properties and methods:
 | Period                            | Meaning                                                      |
 |-----------------------------------|--------------------------------------------------------------|
 | `period.first_month`              | First month-length period that includes the start of `period`|
-| `period.last_month`               | Month preceding `period.this_month`                          |
+| `period.last_month`               | Month preceding `period.first_month`                          |
```

Comment:
Is `last_month` really the month **preceding** `first_month` though?!

Reply:
[It is](https://github.com/openfisca/openfisca-core/blob/master/openfisca_core/periods/period_.py#L462).